toolkit.nu

```diff
@@ -233,9 +233,7 @@ export def "check pr" [
 # - `toolkit fmt --check --verbose` on `git commit`
 # - `toolkit fmt --check --verbose` and `toolkit clippy --verbose` on `git push`
 export def setup-git-hooks [] {
-    if $nu.os-info.name == windows {
-        return (print "This git hook isn't available on Windows. Sorry!")
-    }
+
```

Comment:
could you remove these empty newlines between the command definition and the `print`? :yum:

Reply:
thanks :yum: